recipe/meta.yaml

```diff
@@ -10,39 +10,40 @@ source:
   sha256: 4ce9f5a7cb0950699fe08ebc5a463ab4d09ef550c050391a319308a2494f971f
 
 build:
-  skip: true  # [win]
-  number: 4
+  number: 5
 
 outputs:
   - name: {{ name }}
-    script: install_nceplibs_g2c.sh
+    script: build_linux.sh  # [not win]
+    script: build_win.bat  # [win]
     requirements:
       build:
         - {{ compiler('c') }}
-        - cmake
+        - cmake >=3.15
         - pkg-config >=0.21
         - make                  # [not win]
         - ninja                 # [win]
       host:
         - libpng
         - zlib
+        # list libaec twice to conda-smithy doesn't get confused
+        - libaec
         - libaec >=1.0.6
```

Comment:
This looks a little weird but I think this is how to make conda-smithy work right.